### PR TITLE
Add environment variable check to optionally hide request data from logs

### PR DESCRIPTION
**Pull request by brendanrmoore**
In order to hide sensitive data for logs, I added an environment variable check to conditionally hide certain log statements. This affects one line in `slack/api.js` and affects a few log items in `fetch-error.js`.

I tested locally that request data is hidden when the environment variable is set.

Note: my IDE automatically applied the prettier config when I saved. I assume it's ideal to match the prettier spec, so I left those formatting changes in.

### CODE REVIEW

File: api-module-library/slack/api.js [hunk 1]
```diff
@@ -120,7 +120,9 @@ class Api extends OAuth2Requester {
         const parsedResponse = await this.parsedBody(response);
         const { status } = response;
         const { ok, error } = parsedResponse;
-        console.log(parsedResponse);
```
Comment (brendanrmoore) [inline api-module-library/slack/api.js:123]: I just saw another PR removes this on the main branch. https://github.com/friggframework/frigg/pull/298

I think that probably makes more sense for this particular log line.

Comment (brendanrmoore) [reply api-module-library/slack/api.js:123]: Updated the PR to just remove that line instead (matching the PR on `main`)

**Review by MichaelRyanWebber — APPROVED** [review]
Yup, this works for me!

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`@friggframework/api-module-activecampaign@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-activecampaign%400.11.2), [`@friggframework/api-module-airwallex@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-airwallex%400.11.2), [`@friggframework/api-module-attentive@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-attentive%400.11.2), [`@friggframework/api-module-clyde@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-clyde%400.11.2), [`@friggframework/api-module-connectwise@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-connectwise%400.11.2), [`@friggframework/api-module-crossbeam@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-crossbeam%400.11.2), [`@friggframework/api-module-fastspring-iq@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-fastspring-iq%400.11.2), [`@friggframework/api-module-front@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-front%400.11.2), [`@friggframework/api-module-frontify@1.3.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-frontify%401.3.2), [`@friggframework/api-module-google-calendar@0.3.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-google-calendar%400.3.2), [`@friggframework/api-module-google-drive@0.3.3`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-google-drive%400.3.3), [`@friggframework/api-module-gorgias@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-gorgias%400.11.2), [`@friggframework/api-module-hubspot@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-hubspot%400.11.2), [`@friggframework/api-module-huggg@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-huggg%400.11.2), [`@friggframework/api-module-ironclad@0.3.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-ironclad%400.3.2), [`@friggframework/api-module-marketo@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-marketo%400.11.2), [`@friggframework/api-module-microsoft-sharepoint@0.3.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-microsoft-sharepoint%400.3.2), [`@friggframework/api-module-microsoft-teams@0.4.3`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-microsoft-teams%400.4.3), [`@friggframework/api-module-monday@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-monday%400.11.2), [`@friggframework/api-module-netx@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-netx%400.11.2), [`@friggframework/api-module-outreach@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-outreach%400.11.2), [`@friggframework/api-module-personio@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-personio%400.11.2), [`@friggframework/api-module-pipedrive@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-pipedrive%400.11.2), [`@friggframework/api-module-qbo@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-qbo%400.11.2), [`@friggframework/api-module-rev-io@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-rev-io%400.11.2), [`@friggframework/api-module-rollworks@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-rollworks%400.11.2), [`@friggframework/api-module-salesforce@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-salesforce%400.11.2), [`@friggframework/api-module-salesloft@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-salesloft%400.11.2), [`@friggframework/api-module-slack@0.4.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-slack%400.4.2), [`@friggframework/api-module-terminus@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-terminus%400.11.2), [`@friggframework/api-module-yotpo@0.3.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-yotpo%400.3.2), [`@friggframework/api-module-zoom@0.11.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-zoom%400.11.2), [`@friggframework/assertions@1.0.9`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/assertions%401.0.9), [`@friggframework/core@0.2.15`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/core%400.2.15), [`@friggframework/errors@1.1.7`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/errors%401.1.7), [`@friggframework/integrations@1.2.4`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/integrations%401.2.4), [`@friggframework/module-plugin@1.1.4`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/module-plugin%401.1.4), [`@friggframework/types@0.2.4`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/types%400.2.4) :rocket:

**Pull request merged**